tests/TestHarness/Cluster.py

```diff
@@ -780,6 +780,9 @@ def populateWallet(self, accountsCount, wallet, accountNames: list=None, createP
     def getNodeP2pPort(self, nodeId: int):
         return self.p2pBasePort + nodeId
 
+    def getNodeHttpPort(self, nodeId: int):
+        return self.port + nodeId
```

Comment:
I don't think you need this method. See other comment.

Reply:
Addressed: [f342f5e](https://github.com/AntelopeIO/leap/pull/1138/commits/f342f5e3cace6bd7db29e548640fc9a23fe53d43)

Reply:
This method still here.

Reply:
Fixed.  Accidentally removed it from the build not src version of Cluster.py.

Reply:
[7b41fe2](https://github.com/AntelopeIO/leap/pull/1138/commits/7b41fe2050e7b7709b767b3f5405ffddb19fb314)